### DISCUSSION TITLE
React `useEffect` is not working after exporting a dynamically generated page

### DISCUSSION THREAD

**Discussion by normancapule** [vercel/next.js#13210] [Help]
# Bug report

## Describe the bug

For dynamically generated pages, the logic inside a `useEffect` hook is not being executed. 

This was observed ONLY after exporting the app via `yarn next export`.

When creating a dynamically generated page, we followed these tutorials:
https://nextjs.org/docs/routing/dynamic-routes
https://nextjs.org/docs/basic-features/data-fetching#getstaticpaths-static-generation

## To Reproduce

Please see this repo I created that replicates the bug: https://github.com/normancapule/dynamic-route-next-js-bug

## Expected behavior

`useEffect` should be called even in dynamically generated pages.

## Screenshots

N/A

## System information

- OS: macOS
- Browser (if applies) chrome
- Version of Next.js: 9.4.2
- Version of Node.js: 14.2.0

## Additional context

None

**Comment by danny-fallas** [comment]
This is happening to me. Version 11 fixes it?
